tests/app/celery/test_research_mode_tasks.py

```diff
@@ -37,21 +37,6 @@ def test_make_sns_callback(notify_api, rmock):
     assert 'mobile={}'.format(phone_number) in rmock.request_history[0].text
 
 
-def test_make_pinpoint_callback(notify_api, rmock):
```

Comment:
Wasn't used in tests strangely so deleted that because it has "Pinpoint" in it 👀